### DISCUSSION TITLE
Impersonate user?

### DISCUSSION THREAD

**Discussion by cloudlab-cmu** [Budibase/budibase#8640] [Ideas]
Is there a way I can implement "Impersonation" in budibase?  Basically I would like an admin to be able to login as a particular user and see what that user sees.

**Comment by melohagan** [comment]
Hey cloudlab, 

There is no way to specifically do this, however you could simply create test users with various roles and you would see what any user with that role would see, at least in terms of screens and links.

**Reply by alscheuring** [reply]
Yep thanks. That is what I'm currently doing. I have a small user base that tends to have "trouble" with the system and it would just be nice to see the same thing they are seeing. Thanks for replying!
